<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowCodeEditor</class>
 <widget class="QMainWindow" name="MainWindowCodeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>474</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindowCodeEditor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="horizontalWidget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
         <horstretch>150</horstretch>
         <verstretch>255</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QTreeView" name="fileSystemTreeView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>800</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPlainTextEdit" name="codeEditor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>0</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="tabStopDistance">
        <double>25.000000000000000</double>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalWidget_2" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>235</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>23</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Folder"/>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSession">
    <property name="title">
     <string>Сессии</string>
    </property>
    <addaction name="actionCreateSession"/>
    <addaction name="actionJoinSession"/>
    <addaction name="separator"/>
    <addaction name="actionShowListUsers"/>
    <addaction name="actionCopyId"/>
    <addaction name="actionSaveSession"/>
    <addaction name="actionLeaveSession"/>
   </widget>
   <widget class="QMenu" name="menufd">
    <property name="title">
     <string>Параметры</string>
    </property>
    <addaction name="actionToDoList"/>
    <addaction name="actionTerminal"/>
    <addaction name="actionChangeTheme"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSession"/>
   <addaction name="menufd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_File">
   <property name="text">
    <string>&amp;Новый файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>&amp;Открыть файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="text">
    <string>Открыть &amp;папку</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Сохранить</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Сохранить &amp;как</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Вы&amp;ход</string>
   </property>
  </action>
  <action name="actionCreateSession">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="actionJoinSession">
   <property name="text">
    <string>Вступить</string>
   </property>
  </action>
  <action name="actionShowListUsers">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Список клиентов</string>
   </property>
  </action>
  <action name="actionLeaveSession">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Покинуть</string>
   </property>
  </action>
  <action name="actionSaveSession">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionCopyId">
   <property name="text">
    <string>Скопировать ID</string>
   </property>
  </action>
  <action name="actionChangeTheme">
   <property name="text">
    <string>Сменить тему</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionToDoList">
   <property name="text">
    <string>Открыть ToDo-List</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Терминал</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+`</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
